lnrpc/routerrpc/router_backend_test.go

```diff
@@ -128,6 +135,14 @@ func testQueryRoutes(t *testing.T, useMissionControl bool) {
 			return 1, nil
 		},
 		MissionControl: &mockMissionControl{},
+		FetchChannelEndpoints: func(chanID uint64) (route.Vertex,
+			route.Vertex, error) {
+
+			if chanID != 555 {
+				t.Fatal()
```

Comment:
should the error be returned so it can fail on 150, rather than failing w/o context?

Reply:
The idea here is that we assert that `FetchChannelEndpoints` is called with the correct parameters. Returning this as an error to the caller wouldn't really be what is intended, although it would work too. I've added a string to `t.Fatal` to make it more clear.

Reply:
gotcha i see now, error string lgtm 👌